plugins/modules/azure_rm_aduser_info.py

```diff
@@ -192,6 +204,8 @@ def __init__(self):
         self.all = None
         self.log_path = None
         self.log_mode = None
+        self.given_name = None
+        self.surname = None
```

Comment:
```suggestion
```

Reply:
- name: Get Aduser info and filter by surname
  azure_rm_aduser_info:   
      attribute_name: surname
      attribute_value: *****
      all: True